### ISSUE TITLE
Getting error when installing

### ISSUE THREAD

**Issue by slightlytyler** [danielterwiel/prettier-eslint-webpack-plugin#1]
```
error An unexpected error occurred: "https://registry.yarnpkg.com/prettier-eslint-webpack-plugin/-/prettier-eslint-webpack-plugin-0.1.1.tgz: invalid tar file".
```

I republished the module as `@slightlytyler/prettier-eslint-webpack-plugin` using `npm publish` and it works. A bit of googling points to `yarn publish` being the culprit, might need to republish with npm.

**Comment by danielterwiel** [comment]
Thanks a lot for pointing this out. As I used `npm link` this has gone unnoticed, but surely was worth testing before publishing it 🙈. I published a new version. This time through npm in stead of yarn 🙂.
